packages/neos-ui/src/Containers/Modals/SelectNodeType/index.js

```diff
@@ -61,25 +63,37 @@ export default class SelectNodeType extends PureComponent {
 
     componentDidMount() {
         this.setState({
+            filterSearchTerm: '',
```

Comment:
As said, the default value for `filterSearchTerm` should be set in the constructor